data-imports/clean_utils.py

```diff
@@ -1,5 +1,6 @@
 import os
 from utils import connect
+import os
```

Comment:
We already import os? Seems like all the changes in this file can be reverted, honestly. (Sorry for all the comments.)